kastenwesen

```diff
@@ -1 +0,0 @@
-kastenwesen.py
```

Comment:
I fear that removing this symlink will break using `kastenwesen` from the command line. Can you please make sure that the Vagrantfile still works?